project/plugins.sbt

```diff
@@ -1,2 +1,3 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.25")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
Does it work with this version? I thought it requires `0.10.0-RC1`.

Reply:
I think that I re-used the setup of Analytics SDK where Snyk was already integrated, but actually now that you say that I realize that I forgot to check that it was working when I released new Analytics SDK version, and it didn't : https://github.com/snowplow/snowplow-scala-analytics-sdk/runs/1375024814?check_suite_focus=true
I'll get back to you on this

Reply:
FYI it seems to be running smoothly with `"0.9.2"`